frontend/simple/views/components/Voting.vue

```diff
@@ -96,6 +96,7 @@
   }
 
   &-sign {
+    position: relative;
```

Comment:
@greg On Voting PR, the bug you shown on SVG circle on Safari, what's the version? Because on mine it looks good, also on Firefox, still, I have a slight idea what might cause the bug so I updated the code to fix. Please confirm if it's okay, since I can see it myself xD